s30157-MikolajMach/zadania/zjazd_4.txt

```diff
@@ -0,0 +1,157 @@
+UKOS ZJAZD NR.4
+
+Zadanie 0 (rozgrzewkowe)
+Przypuscmy, ze nie mozesz utworzyc zadnego pliku na swoim koncie uczelnianym
+np. na szuflandii. Co sie moglo stac? (wymien co najmniej 3 opcje)
+Zakladamy, ze mozesz zalogowac sie w konsoli i nie masz zaleglosci w oplatach.
+
+Odpowiedź:
+ -Utracone uprawnienia
+ -Problem z serwerem
+
+
+
+Zadanie 1
+Jest dane takie drzewo katalogow:
+.
+├── ala
+│   ├── i
+│   │   └── as
+│   └── ma
+│       └── kota
+└── kot
+    └── ma
+        └── ale
+
+Znajdujesz sie w katalogu kota. Katalog ala jest w katalogu /tmp/ukos.
+Jak przejsc do katalogu ala za pomoca:
+ - sciezki wzglednej (relatywnej)
+ - sciezki bezwzglednej (absolutnej)
+
+
+Wykonanie:
+ Zeby dostać się do katalogu przy uzyciu sciezki wzglednej wykorzystuje komende: cd ../..
+ Zeby dostać się do katalogu przy uzyciu sciezki bezwzglednej wykorzystuje komende: cd /tmp/ukos/ala
+
+
+
+
+Zadanie 2
+Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak
+utworzyc  poddrzewo katalogow jan/kowalski w katalogu ale za pomoca jednej
+komendy? Podaj rozwiazania uzywajac sciezek absolutnych i relatywnych.
+
+
+Wykonanie:
+ Dla sciezka relatywna wykonuje komende: mkdir -p ../../../kot/ma/ale/jan/kowalski 
+ Dla sciezka absolutna wykonuje komende: mkdir -p /tmp/ukos/kot/ma/ale/jan/kowalski
+
+
+
+
+Zadanie 3
+Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak
+przeniesc katalog ale do katalogu i uzywajac:
+ - zrodlo (sciezka wzgledna), miejsce docelowe (sciezka absolutna)
+ - zrodlo (sciezka absolutna), miejsce docelowe (sciezka wzgledna)
+
+
+Wykonanie:
+ ze sciezki wzglednej do sciezki absolutnej: mv ../../../kot/ma/ale /tmp/ukos/ala/i
+ ze sciezki absolutnej do sciezki wzglednej: mv /tmp/ukos/kot/ma/ale ../../i
+
+
+
+
+
+Zadanie 4
+Jak zamknac program, ktory nie reaguje na ctrl+c?
+
+
+Odpowiedz:
+Zeby zamknac program, wykorzystalbym skrot klawiszowy "CTRL + \"
```

Comment:
Ciekawe. Dobrze ale nikt takiej odpowiedzi jeszcze nie podał. Ładnie.
To teraz jeszcze uzupełnij skąd pomysł na takiego rozwiązanie i co ono de facto zrobi.